### PR TITLE
Remove tqdm, implement custom progress display

### PR DESCRIPTION
**Pull request by cdebourcy**

### CODE REVIEW

File: idseq/uploader.py [hunk 1]
```diff
@@ -258,14 +260,34 @@ def get_user_agreement():
...
     def __init__(self, file_path, i, count):
         super(Tqio, self).__init__(io.open(file_path, "rb"))
-        desc = "%s (%d/%d)" % (file_path, i + 1, count)
-        self.tqdm = tqdm.tqdm(
-            desc=desc,
-            unit="bytes",
-            unit_scale=True,
-            total=os.path.getsize(file_path))
+        curses.initscr()
+        curses.curs_set(0)
```
Comment (boris-dimitrov) [inline idseq/uploader.py]: Does this require a terminal?   Will it work through ssh session that has no interactive terminal attached?

Comment (boris-dimitrov) [reply idseq/uploader.py]: In my experience terminal screen stuff is hard to debug when it fails.

**Comment by boris-dimitrov** [comment]
Also for operational stuff like the percents, logging to stderr rather than stdout may be better.

**Comment by cdebourcy** [comment]
ok removed curses

**Pull request merged**